src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {Button, ButtonProps} from './Button';
+
+const setupComponent = (render: Function, props: {} = {}) => {
```

Comment:
do you need to pass `render` function? As i can see you used `shallow` for both cases

Reply:
Thanks, this is fixed